node-fs-writefile/cp.js

```diff
@@ -0,0 +1,21 @@
+import * as fs from 'node:fs/promises';
+
+const from = process.argv[2];
+const to = process.argv[3];
+
+async function readAndWrite(fromUrl, toUrl) {
+  try {
+    const data = await fs.readFile(fromUrl, 'utf8');
+    await fs.writeFile(toUrl, data);
+  } catch (error) {
+    console.log('ERROR: ', error);
+  }
+}
+
+if (from !== undefined && to !== undefined) {
```

Comment:
As a shortcut you could also say `if (from && to) {...}`.

Sometimes comparing explicitly against `undefined` is necessary, but I try to avoid it since that usually makes the code more robust.